src/test/java/de/tum/in/www1/artemis/assessment/ExampleSubmissionIntegrationTest.java

```diff
@@ -333,8 +333,6 @@ void importExampleSubmissionWithTextSubmission() throws Exception {
         submission = textExerciseUtilService.saveTextSubmission(textExercise, submission, TEST_PREFIX + "student1");
 
         TextBlock textBlock = new TextBlock();
-        textBlock.setCluster(null);
-        textBlock.setAddedDistance(0);
```

Comment:
I understand why we remove the setCluster, but why do we also remove the setAddedDistance?

Reply:
The `addedDistance` of a text block is related to its cluster and is now also stored in Athena directly. It is used for calculating the next suggested submission, which is now a task of the Athena service -> Artemis does not need this any more.